bundles/binding/org.openhab.binding.myq/src/main/java/org/openhab/binding/myq/internal/MyqBinding.java

```diff
@@ -316,6 +316,19 @@ private void computeCommandForItem(Command command, String itemName) {
                 } else if (command.equals(OnOffType.OFF) || command.equals(UpDownType.DOWN)) {
                     myqOnlineData.executeGarageDoorCommand(garageopener.getDeviceId(), 0);
                     beginRapidPoll(true);
+                } else if (command instanceof StringType) {
+                    String stringValue = ((StringType) command).toString();
+                    if (stringValue.equalsIgnoreCase(GarageDoorStatus.OPEN.getLabel())
+                            || stringValue.equalsIgnoreCase(GarageDoorStatus.OPENING.getLabel())) {
+                        myqOnlineData.executeGarageDoorCommand(garageopener.getDeviceId(), 1);
+                        beginRapidPoll(true);
+                    } else if (stringValue.equalsIgnoreCase(GarageDoorStatus.CLOSED.getLabel())
+                            || stringValue.equalsIgnoreCase(GarageDoorStatus.CLOSING.getLabel())) {
+                        myqOnlineData.executeGarageDoorCommand(garageopener.getDeviceId(), 0);
+                        beginRapidPoll(true);
+                    } else {
+                        logger.warn("Unknown string command {}", stringValue);
+                    }
```

Comment:
It makes sense to me to have the commands with the same values that a String item is set to when bound to the door. Doing it this way, you can use a Selection in the sitemap that is used to both display the current state and set the target state.